extension-iac/src/iac_android.cpp

```diff
@@ -178,16 +90,21 @@ int IAC_PlatformSetListener(lua_State* L)
 }
 
 
-static void QueueCommand(IACCommand* cmd)
+
+static void HandleInvocation(const IACCommand* cmd)
 {
-    DM_MUTEX_SCOPED_LOCK(g_IAC.m_Mutex);
-    if (g_IAC.m_CmdQueue.Full())
+    if (!g_IAC.m_Listener)
+    {
+        dmLogError("No iac listener set. Invocation discarded.");
```

Comment:
It isn't exactly discarded is it?

Reply:
This is very true. That line of logging is from the original code. I think I'll remove it completely as I don't see how it would help.